content/docs/sidebar.json

```diff
@@ -128,36 +146,26 @@
           "running-experiments",
           "hydra-composition",
           "comparing-experiments",
+          "visualizing-plots",
```

Comment:
> Possibly follow-up

Should the [Parallel Coordinates Plot](https://dvc.org/doc/user-guide/experiment-management/comparing-experiments#parallel-coordinates-plot) section be moved or linked from the [Visualizing Plots](https://dvc-org-ug-reorg-zcajwdgw5kgni.herokuapp.com/doc/user-guide/experiment-management/visualizing-plots) page?

Reply:
I don't think it should be moved since it's more related to comparing experiments, but linking could make sense.